rasa_core/policies/form_policy.py

```diff
@@ -19,12 +18,14 @@ class FormPolicy(MemoizationPolicy):
 
     def __init__(self,
                  featurizer: Optional[TrackerFeaturizer] = None,
+                 priority: int = 4,
```

Comment:
why does the fallback persist the priority but the form does not?

Reply:
form_policy doesn't have its own persist method since its a subclass of memo, it persists via memo's persist method, which persists priority. With regards to memo's persisting, I wasn't sure if I should create a new meta file to parallel the others, but it seemed like metadata was already being persisted via the `data`  file